src/read.c

```diff
@@ -714,6 +714,11 @@ typedef struct avifMeta
     // and are then further modified/updated as new information for an item's ID is parsed.
     avifDecoderItemArray items;
 
+    // Used when the colorItem is a grid and the alpha item is represented as a set of auxl items
+    // to each color tile. The data of this item need not be read and the pixi property cannot be
+    // validated.
```

Comment:
Document that `alphaItem->id` is 0.

IMPORTANT: Ideally `alphaItem->id` should not be used because it is an invalid value. But it is used at line 4427 in src/read.c:

```
    for (uint32_t i = 0; i < alphaItemCount; ++i) {
        avifDecoderItem * item = meta->items.item[alphaItemIndices[i]];
        item->dimgForID = (*alphaItem)->id;  <=== USED HERE!
    }
```

This can lead to bugs. Note that `dimgForID` is documented as follows:

```
    uint32_t dimgForID;            // if non-zero, this item is an input of derived Item #{dimgForID}
```

Reply:
Outdated code